templates/tftmpl/monitor_catalog_services.go

```diff
@@ -27,6 +27,10 @@ func (m CatalogServicesMonitor) ServicesAppended() bool {
 	return false
 }
 
+func (m CatalogServicesMonitor) SourceIncludesVariable() bool {
```

Comment:
Will add comment:

`SourceIncludesVariable returns whether or not to include the source variables in the template. For the case of a monitor, this always returns true and must be overridden to return based on other conditions.`